q2_fondue/sequences.py

```diff
@@ -88,6 +94,7 @@ def _run_fasterq_dump_for_all(
         accession_ids (list): List of all run IDs to be fetched.
         tmpdirname (str): Name of temporary directory to store the data.
         threads (int, default=1): Number of threads to be used in parallel.
+        key_file (str, default=''): Filepath to dbGaP repository key.
```

Comment:
Actually, the docstring says that `default=''`, but that is not the case when you look at the function definition (same for retries. Maybe you can remove those from the docstring (ideally we would add them to the definition but that would require shuffling the args which is a bit more complicated...).

Reply:
the docstring of `get_sequences` contains the default values that match the function's arguments. Hence, I am simply removing the default values here.